.coveragerc

```
@@ -33,3 +33,5 @@ source =
   .
 source_pkgs =
   multidict
+omit =
+  tests/gen_pickles.py
```

We should probably handle this differently. I've been thinking of refactoring that in a way that wouldn't compromise coverage. There's two PRs exploring this and an issue. But I think that after all, I'd look into delegating this to a pytest plugin.

```suggestion
  # FIXME: migrate to a snapshotting plugin. See #922, #924 and #938.
  tests/gen_pickles.py
```

---

I'd rather suggest moving the generator into `./tools` folder.

---

If a proper pytest plugin is used, we won't even need a separate script. It'd be better integrated.

---

But regardless of that, the ommission deserves a comment with justification.